src/core/jsproxy.c

```diff
@@ -467,9 +467,27 @@ JsProxy_subscript_array(PyObject* o, PyObject* item)
     return pyresult;
   }
   if (PySlice_Check(item)) {
-    PyErr_SetString(PyExc_NotImplementedError,
-                    "Slice subscripting isn't implemented");
-    return NULL;
+    Py_ssize_t start, stop, step, slicelength;
+    if (PySlice_Unpack(item, &start, &stop, &step) < 0) {
```

Comment:
I think we can use `FAIL_IF_MINUS_ONE` here too, same as what you did in `JsArray_ass_subscript`.

Reply:
Can't because there is no `finally` block in this function. Perhaps we should add one that just returns `NULL`.

Reply:
I adulterated my typical C style here with some code that was nearly copied verbatim from `cpython/Objects/listobject.c`